constellations/gemini-admin/src/emails/accountInfo.js

```diff
@@ -0,0 +1,134 @@
+"use strict";
+require("../account/accountInfo.sass");
+import React from "react";
+import { Link } from "react-router-dom";
+import { getFullName } from "../utils/userUtils.js";
+
+export class AccountInfo extends React.Component {
+    state = {
+        id: 0,
+        email: "",
+        users: [],
+        transactions: [],
+        selectedUserEmails: [],
+        active: false,
+    };
+
+    formatCurrency = (amount) => {
+        return new Intl.NumberFormat("en-US", {
+            style: "currency",
+            currency: "USD",
+        }).format(amount);
+    };
+
+    onCheckUser = (e, userId) => {
+        const users = this.props.users;
+        if (e.target.checked) {
+            users.map((user, index) => {
+                if (user.id == userId) {
+                    var email = this.state.selectedUserEmails;
+                    email.push(user.email);
+                    this.setState({
+                        selectedUserEmails: email,
+                    });
+                }
+            });
```

Comment:
Maybe you can use find instead of map here
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
Once you find the target email, then you can edit the list of emails.
Also rename the variable `email` to `emails` to convey a list of emails (instead of one)